### PR TITLE
Update rapidoc and highlight

### PR DESCRIPTION
**Pull request by yubiuser**
**What does this PR aim to accomplish?:**

Updates the shipped `rapidoc` do v3.9.7` which contains a fix for an issue we reported (https://github.com/rapi-doc/RapiDoc/issues/967).

I also took the change to update the embedded `highlight.js` to `11.10.0`

---
**By submitting this pull request, I confirm the following:**

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against.
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
6. I have checked that another pull request for this purpose does not exist.
7. I have considered, and confirmed that this submission will be valuable to others.
8. I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
9. I give this submission freely, and claim no ownership to its content.

---
- [x] I have read the above and my PR is ready for review. *Check this box to confirm*

### CODE REVIEW

**Comment by yubiuser** [comment]
Mhh... we might want to think about this update...

**Comment by yubiuser** [comment]
Also the reported issue still exists

**Comment by yubiuser** [comment]
Ok, I just noted, the first issue is also present in `development`

File: src/api/docs/content/external/highlight.min.js [hunk 1]
```diff
@@ -1,2 +1,400 @@
...
+    relevance:0,contains:["self",a,e.C_BLOCK_COMMENT_MODE,c,o,s]}]
+    },s,a,e.C_BLOCK_COMMENT_MODE,l]};return{name:"C++",
+    aliases:["cc","c++","h++","hpp","hh","hxx","cxx"],keywords:p,illegal:"</",
+    classNameAliases:{"function.dispatch":"built_in"},
+    contains:[].concat(f,g,_,m,[l,{
+    begin:"\\b(deque|list|queue|priority_queue|pair|stack|vector|map|set|bitset|multiset|multimap|unordered_map|unordered_set|unordered_multiset|unordered_multimap|array|tuple|optional|variant|function)\\s*<(?!<)",
+    end:">",keywords:p,contains:["self",s]},{begin:e.IDENT_RE+"::",keywords:p},{
+    match:[/\b(?:enum(?:\s+(?:class|struct))?|class|struct|union)/,/\s+/,/\w+/],
+    className:{1:"keyword",3:"title.class"}}])}}})();hljs.registerLanguage("cpp",e)
+    })();
```
Comment (DL6ER) [inline src/api/docs/content/external/highlight.min.js:400]: Is this really a minified version? The filename suggests so...

Comment (yubiuser) [reply src/api/docs/content/external/highlight.min.js:400]: Yes it is. The full version is even larger

Comment (yubiuser) [reply src/api/docs/content/external/highlight.min.js]: Maybe I force pushed away an error...

**Comment by DL6ER** [comment]
Checked and found no issues. Updating is usually good so let's do it.

**Review by DL6ER — APPROVED** [review]

**Pull request merged**